docs/DeveloperGuide.md

```diff
@@ -1019,6 +1019,57 @@ starting point for testers to work on; testers are expected to do more *explorat
     2. Re-launch the app by double-clicking the jar file. Check the command history again with `history`. <br>
        Expected: The same command history (possibly with one extra `history` and/or `exit` command) is displayed.
 
+### Resident Data
+1. Adding residents
+
+    1. Test case: `radd n/John Doe p/98765432 e/johnd@example.com y/1` <br>
```

Comment:
Hmm maybe specify the prerequisite of there being no existing resident called `John Doe`.